engine/flink/executor/src/test/scala/pl/touk/nussknacker/engine/process/helpers/ProcessTestHelpers.scala

```diff
@@ -116,3 +128,26 @@ object ProcessTestHelpersConfigCreator extends EmptyProcessConfigCreator {
   }
 
 }
+
+case object SkewedMonitorEmptySink extends EmptySink with Serializable {
+
+  val invocationsCount = new AtomicInteger(0)
+
+  private var skewValue: Int = _
+
+  def clear(): Unit = {
+    invocationsCount.set(0)
+  }
+
+  override def valueFunction(
+      helper: FlinkLazyParameterFunctionHelper
+  ): FlatMapFunction[Context, ValueWithContext[AnyRef]] = (_, _) => {
+    invocationsCount.getAndAdd(skewValue)
+  }
+
+  override def toFlinkFunction(flinkNodeContext: FlinkCustomNodeContext): SinkFunction[AnyRef] = {
+    skewValue = flinkNodeContext.nodeId.length
```

Comment:
Why not just save `nodeId` instead of `nodeId.length`? Why skew? Let's make it as simple as possible to make important things (access to `flinkNodeContext`) exposed